### PR TITLE
Merge the staff handbook

### PR DESCRIPTION
**Pull request by demkodo**
updated trailing white spaces, should be good now :)

-----
[View rendered activities/staff-management/index.md](https://github.com/publiccodenet/about/blob/demkodo-patch-2/activities/staff-management/index.md)
[View rendered activities/staff-management/leave.md](https://github.com/publiccodenet/about/blob/demkodo-patch-2/activities/staff-management/leave.md)
[View rendered activities/staff-management/parental-leave.md](https://github.com/publiccodenet/about/blob/demkodo-patch-2/activities/staff-management/parental-leave.md)
[View rendered activities/staff-management/staff-and-contractors.md](https://github.com/publiccodenet/about/blob/demkodo-patch-2/activities/staff-management/staff-and-contractors.md)
[View rendered activities/staff-management/working-days.md](https://github.com/publiccodenet/about/blob/demkodo-patch-2/activities/staff-management/working-days.md)

### CODE REVIEW

File: activities/staff-management/index.md [hunk 1]
```diff
@@ -4,13 +4,37 @@ type: Index
...
+* [Statutory holidays](/organization/yearly-schedule.md)
 * [How to report illness](sickness.md)
-* [How to request reimbursement for expenses](expense.md)
-* [How to travel](travel.md)
 * [How to work remotely](remote-working.md)
+* [Parental leave](parental-leave.md)
+
+## Personnel and recruitment
+
+* [Staff and contractors](staff-and-contractors.md)
```
[the remark this answers is not in the document]
Comment (demkodo) [reply activities/staff-management/index.md]: Added definition contractor, did not change the .md slug of the page, maybe I should?

**Comment by demkodo** [comment]
Hi B, I think I've updated all your requests / feedback. Let me know if that isn't the case

File: activities/staff-management/parental-leave.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+### Paternity leave
+
+You are entitled to 1 week of partner or paternity leave after your partner gives birth.
+
+### More information
+
+[More information about maternity and partner / paternity leave can be found here:](https://business.gov.nl/regulation/leave-schemes/)
+
+If you have questions about parental leave, please reach out to HR <hr@publiccode.net>.
```
Comment (bvhme) [inline activities/staff-management/parental-leave.md]: What I'm missing here, and what should be leading is 'how do you do it'. As a guide it should set out what you need to do if you or your partner is pregnant.

File: glossary/staff-definition.md [hunk 3]
```diff
@@ -5,3 +5,9 @@ type: Resource
 # Staff
 
 Someone employed directly by the Foundation For Public Code. This excludes community members, volunteers and contractors.
+
+# Contractors
+
+Some roles in our organization are fulfilled by contractors. For transparency, these are mentioned with the legal 
+entity they bill through.
```
Comment (bvhme) [inline glossary/staff-definition.md]: I think this needs it's own file as it is a term we use separately from staff, so a `contractor-definition.md` in this folder with this definition in it.

File: activities/staff-management/parental-leave.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
+---
+type: Guide
+explains: how parental leave works. links to the government's website for detailed information
```
Comment (bvhme) [inline activities/staff-management/parental-leave.md]: This should be one sentence, following the explains logically

```suggestion
explains: what to do and what to know when you or your partner are expecting
```

File: activities/staff-management/parental-leave.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+type: Guide
+explains: what to do and what to know when you or your partner are expecting
+---
+
+# Maternity and paternity leave
+
+Pregnant employees are entitled to 4-6 weeks pregnancy leave (before the due date)
+and at least 10 weeks maternity leave (after childbirth).
+
+## Paternity leave
```
[the remark this answers is not in the document]
Comment (demkodo) [reply activities/staff-management/parental-leave.md]: Done!

File: activities/staff-management/parental-leave.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
+---
+type: Guide
+explains: what to do and what to know when you or your partner are expecting
+---
+
+# Maternity and paternity leave
```
Comment (ElenaFdR) [inline activities/staff-management/parental-leave.md:6]:
```suggestion
# Maternity, partner/paternity and parental leave
```

File: activities/staff-management/parental-leave.md [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+
```
Comment (ElenaFdR) [inline activities/staff-management/parental-leave.md:7]: ## Maternity leave

File: activities/staff-management/parental-leave.md [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+Pregnant employees are entitled to 4-6 weeks pregnancy leave (before the due date)
+and at least 10 weeks maternity leave (after childbirth).
+
+## Parental leave
```
Comment (ElenaFdR) [inline activities/staff-management/parental-leave.md:12]:
```suggestion
## Partner/paternity leave
```

File: activities/staff-management/parental-leave.md [hunk 9]
```diff
@@ -0,0 +1,26 @@
...
+
+You are entitled to 1 week of parental leave after your partner gives birth.
```
Comment (ElenaFdR) [inline activities/staff-management/parental-leave.md:14]:
```suggestion
You are entitled to 1 week of partner/paternity leave after your partner gives birth.
```

File: activities/staff-management/parental-leave.md [hunk 10]
```diff
@@ -0,0 +1,26 @@
...
+### How to
+
+If you are pregnant you have to let HR know at least 3 weeks before you want to go on maternity
+leave. But, the sooner you report it, the better, leaving enough time to make any
+agreements about your absence, and plan a handover of your tasks. For paternity leave, let us know
+when your partner has given birth by 9am the next working day.
+
+#### More information
+
+[More information about maternity and partner / paternity leave can be found here:](https://business.gov.nl/regulation/leave-schemes/)
```
Comment (ElenaFdR) [inline activities/staff-management/parental-leave.md:25]:
```suggestion
[Read more about maternity and partner/paternity leave](https://business.gov.nl/regulation/leave-schemes/) (on the Dutch government website).
```

**Comment by ElenaFdR** [comment]
Hey, I've made some suggested edits. I was wrong to suggest 'parental leave' as a gender neutral version of maternity/paternity leave, because the Dutch government already uses 'parental leave' to mean something totally different. My previous suggestion would have just added confusion. Hopefully the new suggestions are more faithful to the Dutch gov't's website.

**Comment by demkodo** [comment]
I committed your suggestions, thank you :)

File: activities/staff-management/parental-leave.md [hunk 11]
```diff
@@ -0,0 +1,26 @@
+---
+type: Guide
+explains: what to do and what to know when you or your partner are expecting
+---
+
+# Maternity, partner/paternity and parental leave
+
+Pregnant employees are entitled to 4-6 weeks pregnancy leave (before the due date)
+and at least 10 weeks maternity leave (after childbirth).
```
Comment (bvhme) [inline activities/staff-management/parental-leave.md:9]: As per the previous suggestion

```suggestion
and at least 10 weeks birth leave.
```

File: activities/staff-management/parental-leave.md [hunk 12]
```diff
@@ -0,0 +1,26 @@
...
+
+## Partner/paternity leave
+
+You are entitled to 1 week of partner/paternity leave after your partner gives birth.
+
+### How to
+
+If you are pregnant you have to let HR know at least 3 weeks before you want to go on maternity
```
Comment (bvhme) [inline activities/staff-management/parental-leave.md:17]:
```suggestion
If you are pregnant you have to let HR know at least 3 weeks before you want to go on pregnancy
```

File: activities/staff-management/parental-leave.md [hunk 13]
```diff
@@ -0,0 +1,26 @@
...
+leave. But, the sooner you report it, the better, leaving enough time to make any
+agreements about your absence, and plan a handover of your tasks. For paternity leave, let us know
```
Comment (bvhme) [inline activities/staff-management/parental-leave.md:19]:
```suggestion
agreements about your absence, and plan a handover of your tasks. For partner leave, let us know
```

File: activities/staff-management/index.md [hunk 14]
```diff
@@ -4,13 +4,29 @@ type: Index
...
+
+When we expect you to be here and how to plan to be somewhere else.
 
 * [How to request leave](leave.md)
+* [Statutory holidays](../../organization/yearly-schedule.md)
 * [How to report illness](sickness.md)
-* [How to request reimbursement for expenses](expense.md)
-* [How to travel](travel.md)
 * [How to work remotely](remote-working.md)
+* [Parental leave](parental-leave.md)
```
Comment (bvhme) [inline activities/staff-management/index.md:17]:
```suggestion
* [Pregnancy, birth, and partner leave](parental-leave.md)
```

File: activities/staff-management/parental-leave.md [hunk 15]
```diff
@@ -0,0 +1,26 @@
...
+when your partner has given birth by 9am the next working day.
+
+#### More information
+
+[Read more about maternity and partner/paternity leave](https://business.gov.nl/regulation/leave-schemes/) (on the Dutch government website).
```
Comment (bvhme) [inline activities/staff-management/parental-leave.md:24]:
```suggestion
* [Zwangerschapsverlof en bevallingsverlof (Dutch)](https://www.rijksoverheid.nl/onderwerpen/zwangerschapsverlof-en-bevallingsverlof) on rijksoverheid.nl
* [Maternity and partner/paternity leave](https://business.gov.nl/regulation/leave-schemes/) on business.gov.nl.
```

**Review by bvhme — APPROVED** [review]
I approve this!

I have some suggestions for making the content around pregnancy a bit more trans inclusionary.

File: activities/staff-management/leave.md [hunk 16]
```diff
@@ -45,3 +45,11 @@ Emergency leave is paid leave and is separate from vacation.
 Please ask your manager for emergency leave as soon as possible (and not longer than 24 hours after leaving) in order to discuss the terms.
 
 Emergency leave is not requested or logged through Tentoo NMBRS ESS (employee self service).
+
+## Short-term care leave
+
+Short-term care leave can be taken to provide essential care to parents, ill children who still live at home or partners. However, this leave is only granted on the condition if you are the only person who can look after the ill person at that moment in time. During the period of leave, you receive 70% of your salary.
```
Comment (ElenaFdR) [inline activities/staff-management/leave.md:51]:
```suggestion
Short-term care leave can be taken to provide essential care to ill parents, children who still live at home or partners. However, this leave is only granted on the condition that you are the only person who can look after the ill person at that moment in time. During the period of leave, you receive 70% of your salary.
```

File: activities/staff-management/parental-leave.md [hunk 17]
```diff
@@ -0,0 +1,27 @@
...
+leave. But, the sooner you report it, the better, leaving enough time to make any
+agreements about your absence, and plan a handover of your tasks. For partner leave, let us know
+when your partner has given birth by 9am the next working day.
+
+#### More information
+
+* [Zwangerschapsverlof en bevallingsverlof (Dutch)](https://www.rijksoverheid.nl/onderwerpen/zwangerschapsverlof-en-bevallingsverlof) on rijksoverheid.nl
+* [Maternity and partner/paternity leave](https://business.gov.nl/regulation/leave-schemes/) on business.gov.nl.
+
+If you have questions about parental leave, or need to take parental leave, please reach out to HR <hr@publiccode.net>
```
Comment (ElenaFdR) [inline activities/staff-management/parental-leave.md:27]:
```suggestion
If you have questions about parental leave, or need to take parental leave, please reach out to HR <hr@publiccode.net>.
```

**Review by ElenaFdR — APPROVED** [review]
Let's merge this!

File: activities/tool-management/mailing-lists.md [hunk 18]
```diff
@@ -45,7 +45,7 @@ You can also unsubscribe via the web page by [simply clicking on the mailing lis
 
 ## Guidance for mailing list administrators
 
-You should make sure potential and current mailing list members have easy access to the guidance for people who join our mailing lists.
+You should make sure potential and current mailing list members have easy access to the guidance for people who join our mailing lists. The Operations coordinator can assign list owner status to other team members. With this status team members can sign themselves up to create an account on our Mailman instance. With the account the control panel can be accessed to update and customize list settings, as well as monitor pending confirmations from newly signed up members that may not have received the confirmation mail. This mail may end up in spam folders which we can notify them about.
```
Comment (ElenaFdR) [inline activities/tool-management/mailing-lists.md:48]:
```suggestion
You should make sure potential and current mailing list members have easy access to the guidance for people who join our mailing lists. 

The Operations coordinator can assign list owner status to other team members. With this status team members can sign themselves up to create an account on our Mailman instance. With a list owner account, the control panel can be accessed to update and customize list settings, as well as monitor pending confirmations from newly signed up members that may not have received the confirmation mail. This mail may end up in spam folders which we can notify them about.
```

**Review by ElenaFdR — APPROVED** [review]
Ready to go, including Tentoo update!

**Comment by ElenaFdR** [comment]
This closes #761.

**Pull request merged**